jbi/actions/steps.py

```diff
@@ -198,3 +196,37 @@ def maybe_update_issue_status(context: ActionContext, **parameters):
             return context, (resp,)
 
     return context, ()
+
+
+def maybe_update_components(context: ActionContext, **parameters):
+    """
+    Update the Jira issue components
+    """
+    config_components: list[str] = parameters.get("jira_components", [])
```

Comment:
I wonder if there's a better way to handle this than casting. What if someone makes `jira_components` a comma-delimited string, for example. Hopefully something we'd catch in code review for config changes, but something could slip through.

We could maybe type the parameters object or include a guard to check that `jira_components` is a list, emitting a warning if it isn't.

Reply:
That's already how it's done on `main`. I opened https://github.com/mozilla/jira-bugzilla-integration/pull/523 for that